website/assets/site.css

```
@@ -27,46 +28,25 @@ body {
 
 body {
   min-height: 100vh;
-  background: radial-gradient(1200px 600px at 18% -10%, rgba(125, 164, 255, 0.18), transparent 56%),
-    radial-gradient(900px 520px at 88% -8%, rgba(85, 210, 255, 0.16), transparent 58%),
-    var(--bg);
+  background: var(--bg);
   color: var(--text);
   font-family: "Manrope", "Avenir Next", "Segoe UI", sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)